imap_processing/cdf/config/shared/hermes_default_global_cdf_attrs_schema.yaml

```diff
@@ -0,0 +1,278 @@
+  default: null
+  derived: false
+  required: true # NOT Required in ISTP Guide (Recommended)
+  validate: true
+  overwrite: false
+Data_level:
+  description: >
+    This attribute is used in file name creation and records the level of processsing done
+    on the dataset. For HERMES the following are valid values:
+    - l0>Level 0
+    - l1>Level 1
+    - l2>Level 2
+    - l3>Level 3
+    - l4>Level 4
+    - ql>Quicklook
+  default: null
+  derived: false
+  required: true # NOT Required in ISTP Guide (Derived)
+  validate: false
+  overwrite: true
+Data_product_descriptor:
+  description: > 
+    This is an optional field that may not be needed for all products. Where it is used, identifier
+    should be short (e.q. 3-8 characters) descriptors that are helpful to end- users. If a 
+    descriptor contains multiple components, underscores are used to separate those components.
+  default: null
+  derived: false
+  required: false # NOT Required in ISTP Guide (Derived)
+  validate: false
+  overwrite: true
+Data_type:
+  description: >
+    This attribute is used by CDF file writing software to create a filename. It is a
+    combination of the following filename components: mode, data level, and optional data
+    product descriptor.
+  default: null
+  derived: true
+  required: true # NOT Required in ISTP Guide (Derived)
+  validate: false
+  overwrite: true
+Data_version:
+  description: >
+    This attribute identifies the version (vX.Y.Z) of a particular CDF data file.
+  default: null
+  derived: false
+  required: true
+  validate: true
+  overwrite: false
+Descriptor:
+  description: >
+    This attribute identifies the name of the instrument or sensor that collected the data. Both
+    a long name and a short name are given. For any data file, only a single value is allowed.
+    For HERMES, the following are valid values:
+    - EEA>Electron Electrostatic Analyzer
+    - MERIT>Miniaturized Electron pRoton Telescope
+    - NEMISIS> Noise Eliminating Magnetometer In a Small Integrated System
+    - SPAN-I>Solar Probe Analyzer for Ions
+  default: null
+  derived: false
+  required: true
+  validate: true
+  overwrite: false
+Discipline:
+  description: >
+    This attribute describes both the science discipline and sub discipline. For HERMES,
+    this value should always be "Space Physics>Magnetospheric Science."
+  default: Space Physics>Magnetospheric Science
+  derived: false
+  required: true
+  validate: true
+  overwrite: false
+Generation_date:
+  description: >
+    Date stamps the creation of the file using the syntax yyyymmdd, e.g., "
+  default: null
+  derived: true
+  required: true # NOT Required in ISTP Guide (Recommended)
+  validate: true
+  overwrite: true
+HERMES_version:
+  description: >
+    Version of `hermes_core` originally used to generate the given CDF File
+  default: null
+  derived: true
+  required: true # NOT Required in ISTP Guide (Derived)
+  validate: false
+  overwrite: false
```

Comment:
It might be better to have an `imap_processing_version` attribute here? This is so we can tell which version of our software was used to generate different data products.

Reply:
That's something that should go into a separate IMAP global CDF schema (not yet created)